CMake does not include ZeroMQ include path

My CMake file follows the instructions set out by the README.md. This says that it will include ZeroMQ for you, but I have found the following: 

```
find_package(cppzmq)
if(cppzmq_FOUND)
    message(STATUS "ZeroMQ Include Dir: " ${ZeroMQ_INCLUDE_DIR} )
    message(STATUS "C++ZMQ Include Dir: " ${cppzmq_INCLUDE_DIR} )
    include_directories(${cppzmq_INCLUDE_DIR})
    message(STATUS "C++ZMQ Library: " ${cppzmq_LIBRARY} )
    target_link_libraries(server ${cppzmq_LIBRARY})
endif()
```

Gives:
-- ZeroMQ Include Dir: C:/Program Files (x86)/ZeroMQ/include
-- C++ZMQ Include Dir: C:/Program Files (x86)/cppzmq/include
-- C++ZMQ Library: C:/Program Files (x86)/ZeroMQ/bin/libzmq-v141-mt-4_3_1.dll

The build fails with the following error message:
c:\program files (x86)\cppzmq\include\zmq.hpp(43): fatal error C1083: Cannot open include file: 'zmq.h': No such file or directory

The package finds ZeroMQ include directories, but does not include them.

--- a/README.md
+++ b/README.md
@@ -28,7 +28,7 @@ cpp zmq (which will also include libzmq for you).
 #find cppzmq wrapper, installed by make of cppzmq
 find_package(cppzmq)
 if(cppzmq_FOUND)
-    include_directories(${cppzmq_INCLUDE_DIR})
+    include_directories(${ZeroMQ_INCLUDE_DIR} ${cppzmq_INCLUDE_DIR})
     target_link_libraries(*Your Project Name* ${cppzmq_LIBRARY})
 endif()
 ```

